### PR TITLE
Unshelled Hickory Nuts Typo

### PR DESCRIPTION
**Pull request by RogueYun**
My attempt to fix a typo that has been bothering me. When something is "unshelled" it means that it is still in its husk/shell.  It is derived from the verb "to shell" meaning "to remove the shell or outer covering of (something)" http://www.merriam-webster.com/dictionary/shell.

This was also a learning experience for me, so this should be reviewed thoroughly before implementation.

### CODE REVIEW

**Comment by MisterFelixFox** [comment]
Some people (like me) are stupid and didn't know that was the improper usage of the word, but even though 'shelled' is proper it might make people thing they are still in their shells. Is de-shelled a possiblity?

**Comment by Coolthulhu** [comment]
Or maybe "hickory nuts in shells" vs just "hickory nuts".

**Comment by RogueYun** [comment]
The description of the item helps verify what "shelled" means. So I think it is fine. I learn a lot from video games, others can too :D

Perhaps "shucked hickory nuts". But from what I understand shucking the shells from something deals with leverage of some kind and not generally with crushing the shell to remove it... Though do as you will, this was just a learning exercise for me.

**Comment by BevapDin** [comment]
Please don't change the ids of things. This will break saves as they still refer to the old id.

You also don't need to update the translation stuff. This is done periodically by VlasovVitaly.

In short: changing the name and plural name in the json file is enough.

**Comment by chaosvolt** [comment]
Ah, the good old "inflammable/nonflammable" conundrum, in a nutshell. `*`ba dum tish`*`

**Comment by remyroy** [comment]
@RogueYun Awesome initiative! I told you. You can do it :)

**Comment by remyroy** [comment]
@RogueYun To make adjustments based on the comments here, just keep committing in your local branch and push it back in you github account (`git push`). They will appear in this thread (or PR) automatically once that is done.

**Comment by remyroy** [comment]
I think `shelled hickory nuts` is correct. I did not know many of the Japanese swords/blades names before playing this game. I learned by reading the description. This is a great learning opportunity for others as well.

**Comment by MisterFelixFox** [comment]
I retract my earlier statement after seeing remyroy's comment -- I agree that it'll teach people the proper usage (I find myself looking up many things I find ingame as well)

**Comment by RogueYun** [comment]
Thanks everyone for the feedback!
remyroy told me about the item id's before. And I would be sorry for anyone that might have to lose a lot of hours (perhaps days and months) of gameplay. Would it be possible to sit on this for a while and then make the merge at a time when it wouldn't inconvenience anyone? Or at a time where everyone must be inconvenienced anyway?

**Comment by kevingranade** [comment]
Nope, don't change the ids, it doesn't matter if they're spelled right or not.

File: data/json/items/comestibles.json [hunk 1]
```diff
@@ -8142,9 +8142,9 @@
     },
     {
         "type" : "COMESTIBLE",
-        "id": "hickory_nut_unshelled",
-        "name": "handful of unshelled hickory nuts",
-        "name_plural": "handfuls of unshelled hickory nuts",
+        "id": "hickory_nut_shelled",
+        "name": "handful of shelled hickory nuts",
```
Comment (kevingranade) [inline data/json/items/comestibles.json]: The only things you need to (or for that matter the only things you CAN) change are "name" and "name_plural".

**Comment by chaosvolt** [comment]
Technically it's a matter of "shouldn't" than "can't" regarding the ID. One can but then things break.

Swapping their IDs would cause less breakage than if they were changed outright, but that would still result in hilarity ensuing.

**Comment by RogueYun** [comment]
Okie doke! The appropriate changes have been made and it seems to work on my end.

**Pull request merged**